### PR TITLE
done one consumer instead of four

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

**Review by Cheephot — APPROVED** [review]

File: src/ImageService/Startup.cs [hunk 1]
```diff
@@ -178,63 +164,9 @@ private void ConfigureMassTransit(IServiceCollection services)
...
-            {
-                ep.ConfigureConsumer<GetImageProjectServiceConsumer>(context);
-            });
-
-            cfg.ReceiveEndpoint(_rabbitMqConfig.CreateImagesProjectEndpoint, ep =>
-            {
-                ep.ConfigureConsumer<CreateImageProjectServiceConsumer>(context);
-            });
-
             cfg.ReceiveEndpoint(_rabbitMqConfig.DeleteImagesProjectEndpoint, ep =>
```
Comment (ndoriomedov) [inline src/ImageService/Startup.cs:167]: DeleteImagesEndpoint

Comment (VLiamin) [reply src/ImageService/Startup.cs]: done

File: src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+using LT.DigitalOffice.Models.Broker.Enums;
+using LT.DigitalOffice.Models.Broker.Requests.Image;
+using MassTransit;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ImageService.Broker.Consumers
+{
+    public class RemoveImagesConsumer : IConsumer<IRemoveImagesRequest>
+    {
+        private readonly IImageRepository _imageRepository;
```
Comment (MariyaSin) [inline src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs:12]: the consumer has one repository, you can rename variable to _repository

Comment (VLiamin) [reply src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs]: done

File: src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        public async Task Consume(ConsumeContext<IRemoveImagesRequest> context)
+        {
+            object response = OperationResultWrapper.CreateResponse(RemoveImages, context.Message);
+
+            await context.RespondAsync<IOperationResult<bool>>(response);
+        }
+
+        private object RemoveImages(IRemoveImagesRequest request)
+        {
+            switch (request.ImageSource)
```
Comment (MariyaSin) [inline src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs:29]: use it in repo

Comment (VLiamin) [reply src/ImageService.Broker/Consumers/RemoveImagesConsumer.cs]: done

**Review by MariyaSin — APPROVED** [review]

**Review by Glunky — APPROVED** [review]

File: src/ImageService/appsettings.json [hunk 4]
```diff
@@ -48,18 +48,7 @@
...
-    "GetImagesNewsEndpoint": "ImageService_GetImagesNews",
-    "CreateImagesNewsEndpoint": "ImageService_CreateImagesNews",
-    "DeleteImagesNewsEndpoint": "ImageService_DeleteImagesNews",
-    "GetImagesMessageEndpoint": "ImageService_GetImagesMessage",
-    "CreateImagesMessageEndpoint": "ImageService_CreateImagesMessage",
-    "DeleteImagesMessageEndpoint": "ImageService_DeleteImagesMessage",
-    "GetImagesProjectEndpoint": "ImageService_GetImagesProject",
-    "CreateImagesProjectEndpoint": "ImageService_CreateImagesProject",
-    "DeleteImagesProjectEndpoint": "ImageService_DeleteImagesProject"
+    "DeleteImagesEndpoint": "ImageService_DeleteImages"
```
Comment (MariyaSin) [inline src/ImageService/appsettings.json:51]: remove

File: src/ImageService/Startup.cs [hunk 5]
```diff
@@ -178,63 +164,9 @@ private void ConfigureMassTransit(IServiceCollection services)
             IBusRegistrationContext context,
             IRabbitMqBusFactoryConfigurator cfg)
         {
-            cfg.ReceiveEndpoint(_rabbitMqConfig.GetImagesProjectEndpoint, ep =>
+            cfg.ReceiveEndpoint(_rabbitMqConfig.DeleteImagesEndpoint, ep =>
```
Comment (MariyaSin) [inline src/ImageService/Startup.cs:167]: delete => remove

File: src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs [hunk 6]
```diff
@@ -4,17 +4,6 @@ namespace LT.DigitalOffice.ImageService.Models.Dto.Configuration
...
-        public string CreateImagesMessageEndpoint { get; set; }
-        public string GetImagesMessageEndpoint { get; set; }
-        public string DeleteImagesMessageEndpoint { get; set; }
-        public string CreateImagesNewsEndpoint { get; set; }
-        public string GetImagesNewsEndpoint { get; set; }
-        public string DeleteImagesNewsEndpoint { get; set; }
-        public string CreateImagesUserEndpoint { get; set; }
-        public string GetImagesUserEndpoint { get; set; }
-        public string DeleteImagesUserEndpoint { get; set; }
+        public string DeleteImagesEndpoint { get; set; }
```
[the remark this answers is not in the document]
Comment (VLiamin) [reply src/ImageService.Models.Dto/Configuration/RabbitMqConfig.cs]: done

File: src/ImageService/appsettings.json (hunk 4 above)
Comment (VLiamin) [reply src/ImageService/appsettings.json]: done

File: src/ImageService/Startup.cs (hunk 5 above)
Comment (VLiamin) [reply src/ImageService/Startup.cs]: done

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**